geometry_msgs/PointStamped image_click
---

